src/w3d/renderer/ddsfile.cpp

```diff
@@ -45,10 +45,16 @@ DDSFileClass::DDSFileClass(const char *filename, unsigned reduction_factor) :
 {
     // Copy the name across and replace the extension with dds.
     strlcpy_tpl(m_name, filename);
+
+#if 0
     size_t name_len = strlen(m_name);
     m_name[name_len - 3] = 'd';
     m_name[name_len - 2] = 'd';
     m_name[name_len - 1] = 's';
+#else
+    // #BUGFIX Change texture extension safely.
+    Thyme::Change_Texture_File_Extension(m_name, "dds");
```

Comment:
This is pointless. There is absolutely no point for keeping the old code we have git for source control

Reply:
Ok. I removed the ifdef 0.